docs/read.md

```diff
@@ -1,15 +1,59 @@
 # Read
 
-- [Basic Reads](#advanced-reads)
-- [Partial Reads](#partial-reads)
-- [Value Reads](#value-reads)
+- [Basic Reads](#basic-reads)
+  - [Read](#read)
+  - [Partial Reads](#partial-reads)
+  - [Value Reads](#value-reads)
+- [Advanced Reads](#advanced-reads)
+   - [Alias Reads](#alias-reads)
+   - [Derived Data Reads](#derived-data-reads)
 - [Query Syntax](#query-syntax)
-- [useCache](#advanced-reads)
+  - [Collections](#collections)
+  - [Single Document](#single-document)
+  - [Collection Group](#collection-group)
+  - [where](#where)
+  - [Where Clauses](#where-clauses)
+  - [Order By](#orderby)
+  - [Limit](#limit)
+  - [startAt/startAfter](#startatstartafter)
+  - [endAt/endBefore](#endatendbefore)
+- [Firestore Indexes](#firestore-indexes)
+- [Advanced Strategies](#advanced-strategies)
+  - [Rendering lists](#rendering-lists)
+  - [Pagination](#pagination)
+  - [Sliding Windows](#sliding-windows)
+  - [useCache](#usecache)
 
 ---
 
 ## Basic Reads
 
+**Read/Write Subscriptions**
+
+In Read/Write subscriptions are turned on by default. Anytime useRead is
+called it automatically listens for any document changes both locally and
+global and will rerender when data changes. As of v1, there is not a single
```

Comment:
How about this? @comlaterra 
```suggestion
In Read/Write subscriptions are turned on by default. Anytime useRead is
called it automatically listens for any document changes both locally and
globally. When the order changes, docs are added/removed from the query 
results or the contents of a document are changed, then the hook will fire 
and this will trigger the React component to rerender. As of v1, there is not a single
```

Reply:
Yes, that's what I meant :) Thanks